graphql/gqlgen.yml

```diff
@@ -0,0 +1,13 @@
+# .gqlgen.yml example
```

Comment:
I would move this up into the root directory, to keep config files in the same place

Reply:
i would argue that it makes more sense to keep the three files related to gql all in the same directory

Reply:
I think john's right here.

Reply:
meh, sure